### PR TITLE
CC-9276: Support configurable products in the shopping list.

### PR DESCRIPTION
**Pull request by dmiseev**
## PR Description

- Epic: https://spryker.atlassian.net/browse/CC-9276

- Ticket for documentation: https://spryker.atlassian.net/browse/CC-20409

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-product-configuration-glue-api.md [hunk 1]
```diff
@@ -860,7 +866,7 @@ class WishlistsRestApiDependencyProvider extends SprykerWishlistsRestApiDependen
 {% info_block warningBox "Verification" %}
 
 Ensure that wishlist item CRUD operations support configurable products.
-For an example, see the following response to the `POST https://glue.mysprykershop.com/wishlists/{{wishlist_id}}/wishlist-items` request:
+For an example, see the following response to the `POST https://glue.mysprykershop.com/wishlists/63b14493-021f-59c2-ae70-94041beb5c06/wishlist-items` request:
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-product-configuration-glue-api.md:869]: why do we specify exact uuid?
shouldn't we use `{{wishlist_uuid}}` instead?

File: _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-product-configuration-glue-api.md [hunk 2]
```diff
@@ -918,3 +924,146 @@ For an example, see the following response to the `POST https://glue.mysprykersh
...
+        ];
+    }
+}
+
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that wishlist item CRUD operations support configurable products.
+For an example, see the following response to the `POST https://glue.mysprykershop.com/shopping-lists/63b14493-021f-59c2-ae70-94041beb5c06/shopping-list-items` request:
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-product-configuration-glue-api.md:1012]: the same as above

File: _includes/pbc/all/integrate-features/202204.0/integrate-the-product-configuration-feature.md [hunk 3]
```diff
@@ -1245,6 +1254,135 @@ Make sure that the wishlist plugins work correctly:
...
+    {
+        return [
+            new ProductConfigurationShoppingListExpanderPlugin(),
+        ];
+    }
+}
+
+```
+
+**src/Pyz/Client/ShoppingList/ShoppingListDependencyProvider.php**
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/integrate-features/202204.0/integrate-the-product-configuration-feature.md:1316]:
```suggestion
**src/Pyz/Zed/ShoppingList/ShoppingListDependencyProvider.php**
```

File: _includes/pbc/all/integrate-features/202204.0/integrate-the-product-configuration-feature.md [hunk 4]
```diff
@@ -1666,7 +1828,7 @@ console frontend:yves:build
 
 {% info_block warningBox "Verification" %}
 
-Make sure that product configuration data is displayed on the *Product Details*, *Cart* and Wishlist pages.
+Make sure that product configuration data is displayed on the *Product Details*, *Cart*, *Wishlist* and Shopping list pages.
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/integrate-features/202204.0/integrate-the-product-configuration-feature.md:1831]:
```suggestion
Make sure that product configuration data is displayed on the *Product Details*, *Cart*, *Wishlist* and *Shopping list* pages.
```

File: _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-product-configuration-glue-api.md (hunk 1 above)
Comment (dmiseev) [reply _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-product-configuration-glue-api.md:869]: It's needed for consistency (`https://glue.mysprykershop.com/carts/...`)

**Review by DmitryLymarenko — APPROVED** [review]

**Review by DmitryLymarenko — DISMISSED** [review]

**Review by jeremyfourna — DISMISSED** [review]
OK for me

**Review by abitskil — APPROVED** [review]

**Review by DmitryLymarenko — APPROVED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**